manager/integration/tests/test_node.py

```diff
@@ -1170,6 +1170,7 @@ def test_offline_node():
 
     k8s_api_client = get_core_api_client()
     longhorn_api_client = get_longhorn_api_client()
+    cloudprovider = digitalocean()
```

Comment:
Feel we should able to implement it in a better way, so we can choose cloud provider using the environmental setting.